src/components/ListBox/test-helpers.js

```diff
@@ -39,23 +39,33 @@ export const assertMenuClosed = wrapper => {
   );
 };
 
-// `GenericItem` corresponds to an item in a collection that is passed to
-// MultiSelect that is in a predictable shape and works with the default
-// `itemToString` out of the box.
+/**
+ * `GenericItem` corresponds to an item in a collection that is passed to
+ * MultiSelect that is in a predictable shape and works with the default
+ * `itemToString` out of the box.
+ * @param {number} index
```

Comment:
Want to add the `@returns` annotation in these JSDoc blocks as well? 😄

Reply:
For some reason the `return` annotation is not detected for `generateItems` when I do `@returns {Object[]}` or `@returns {Array<Object>}`. I'm using VSCode.

Should I just add `@returns {Array<Object>}`?

Reply:
Done ✔️